bootstrap/kubeadm/internal/bottlerocket/bootstrap.go

```diff
@@ -72,6 +72,17 @@ no-proxy = [{{stringsJoin .NoProxyEndpoints "," }}]
 data = "{{.RegistryMirrorCACert}}"
 trusted=true
 {{- end -}}
+`
+	registryMirrorCredentialsTemplate = `{{define "registryMirrorCredentialsSettings" -}}
+[[settings.container-registry.credentials]]
+registry = "public.ecr.aws"
```

Comment:
Please add a comment here why you need to provide creds for public.ecr.aws. And that we need to remove this once the Bottlerocket fixes are in.